### PR TITLE
GUC-183 add name and description to selfservice client

### PR DESCRIPTION
**Pull request by torsteis**

### CODE REVIEW

File: commands/realm_client.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	realmCmd.AddCommand(realmClientCmd)
+}
+
+var realmClientCmd = &cobra.Command{
+	Use:   "client [realm clientId]",
+	Short: "Get information about a client",
+	Long:  "Get information about a client",
+	Args:  cobra.ExactArgs(2),
+	Run: func(cmd *cobra.Command, args []string) {
+		realm := args[0]
```
Comment (freshteapot) [inline commands/realm_client.go:19]: We already have a top level "keycloak-realm" so my question is, do we need to pass this in here. (At least I dont think)

```
--keycloak-realm string      Realm to use (default "telia")
```

File: commands/realm_client.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+		client := Initialise()
+
+		clientRepresentation, errorMessage := client.KeycloakClient.GetClientRepresentation(realm, clientId)
+		if errorMessage != nil {
+			fmt.Println(errorMessage)
+			return
+		}
+
+		fmt.Printf("ClientId %s has name %s and description %s\n",
+			clientId,
```
Comment (freshteapot) [inline commands/realm_client.go]: I will change this to use the clientRepresentation.Id.

File: commands/root.go [hunk 3]
```diff
@@ -58,6 +58,7 @@ func Initialise() *keycloak.KeycloakOidcClient {
 	provider, err := oidc.NewProvider(ctx, oidcProvider)
 	if err != nil {
 		log.Fatalln("Failed to get an oidc Provider")
+		log.Fatalln("Error %s", err)
```
Comment (freshteapot) [inline commands/root.go]: I will change this to use FatalF

**Review by freshteapot — CHANGES_REQUESTED** [review]

File: commands/realm_client.go (hunk 1 above)
Comment (freshteapot) [reply commands/realm_client.go:19]:
```
$ go run cmd/cli/main.go realm client telia 6086e06a017c4651
ClientId 6086e06a017c4651 has name New self service and description 
```

```
Chriss-MacBook-Pro:divx-taas-keycloak-golang freshteapot$ go run cmd/cli/main.go --keycloak-realm bob realm client telia 6086e06a017c4651
2018/05/23 11:29:30 Failed to get an oidc Provider
exit status 1
```

Comment (torsteis) [reply commands/realm_client.go:19]: I have just tried to mimic what was already done, so if you think we don't need the realm, we go for your solution :)

Comment (freshteapot) [reply commands/realm_client.go:19]: I will fix it up.

**Comment by freshteapot** [comment]
I will fix this after.

**Pull request merged**